tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -340,11 +340,11 @@
     "SampleProjectName": "Samples.MSTestTests",
     "NugetPackageSearchName": "MSTest.TestFramework",
     "MinVersion": "2.0.0",
-    "MaxVersionExclusive": "3.1.0",
+    "MaxVersionExclusive": "4.0.0",
     "SpecificVersions": [
       "2.0.*",
       "2.*.*",
-      "3.0.*"
+      "3.*.*"
```

Comment:
I updated this, as I couldn't see a good reason we shouldn't test 3.1.x versions. Worst case, something breaks, and we know about it. To be honest, I wonder if we should remove the upper bound entirely except where we _explicitly_ don't support the latest in a sample/integration 🤔